test/tests/api/v2_0/nodes_tests.py

```diff
@@ -89,10 +89,10 @@ def __get_workflow_status(self, id):
                 assert_is_not_none(status)
         return status
         
-    def __post_workflow(self, id, graph_name, data):
+    def __post_workflow(self, id, graph_name):
         status = self.__get_workflow_status(id)
         if status != 'pending' and status != 'running':
-            Api().nodes_post_workflow_by_id(identifier=id, name=graph_name, body=data)
+            Api().nodes_post_workflow_by_id(identifier=id, name=graph_name, body={'name': graph_name})
```

Comment:
@benbp We should only need the body, but the generated client library complained when I omitted the name keyword argument.  We need to rebuild the client library and make a minor change to the swagger file to fix this.  I put this PR up as-is to try and get the tests running again, since they are failing most of the time now.